src/Form/FormFactory.php

```diff
@@ -165,6 +171,26 @@ public function createBuilder($key, $data = [], $options = [], $name = null)
         return $formBuilder;
     }
 
+    public function updateFieldOptions(string $type, string $key, array $fieldOptions): array
+    {
+        switch ($type) {
```

Comment:
Do you think this can be used by more fields types in the future and you decided to use a `switch` for that reason instead of an `if` statement?

Reply:
Exactly, I left it like this thinking that in the future more extra functionalities can be added to the field types (used in Linio)